client/css/style.css

```diff
@@ -860,6 +860,10 @@ button,
 	color: #f00;
 }
 
+#chat .msg.toggle .time {
+	visibility: hidden;
+}
+
```

Comment:
Meh, I'm more in favor of moving the right amount of information around instead of too much (like, I remember having lots of redundant objects, not always in sync...), but I guess it will do for now if you think not sending it can't be done in a simpler way.
**This comment comment can now be ignored.**